tests/flow/predicates-declared/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -203,7 +203,7 @@ var n = r(s) || 1;
 (n: number);
 
 var x = \\"\\";
-if (x = r(s) || 1) {
+if ((x = r(s) || 1)) {
```

Comment:
No, the `||` has greater precedence over the assignment so it's evaluated first (or less precedence, whichever one is right. It's the same as `x = (r(s) || 1)`.